backend/globaleaks/models/__init__.py

```diff
@@ -679,8 +728,10 @@ class _Mail(Model):
     address = Column(UnicodeText, nullable=False)
     subject = Column(UnicodeText, nullable=False)
     body = Column(UnicodeText, nullable=False)
+    is_pec = Column(Boolean, nullable=False, default=False)
```

Comment:
TODO: to be renamed secondary_smtp that is the minor fix that wont cause any code rewrite except a variable renaming.

Change needed to abstract the possibility of having a PEC system by enabling to have two SMTP configurations, one primary and one secondary to address the problem of internationalization detailed in https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/4328